include/seastar/testing/perf_tests.hh

```diff
@@ -171,6 +171,7 @@ public:
     [[gnu::always_inline]] [[gnu::hot]]
     void start_run(linux_perf_event* instructions_retired_counter = nullptr) {
         _total_time = { };
+        _total_stats = {};
```

Comment:
I am not sure of the formatting standard for sestar or if there is an auto-formatter for this codebase, but I went with `{}` despite the counter-example of `{ }` on the line immediately because the `{}` is overwhelming more common elsewhere in the seastar codebase.